flutter_local_notifications_windows/lib/src/details/notification_row.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:xml/xml.dart';
+
+import 'notification_part.dart';
+
+/// A group of notification content that must be displayed as a whole row.
+///
+/// See: https://learn.microsoft.com/en-us/uwp/schemas/tiles/toastschema/element-group
+class WindowsRow {
+  /// Makes a group of multiple columns.
+  const WindowsRow(this.columns);
+
+  /// The different columns being grouped together.
+  final List<WindowsColumn> columns;
+
+  /// Serializes this group to XML.
+  void toXml(XmlBuilder builder) => builder.element(
```

Comment:
Can you move this out so it's not part of the public API surface? Could you also see if this could be renamed? It doesn't return a result and methods named `to<...>` conventionally return a result due to how they're named